corehq/apps/export/static/export/js/export_list.js

```diff
@@ -43,6 +43,32 @@ hqDefine("export/js/export_list", [
         });
         options.formname = options.formname || '';
 
+        assertProperties.assert(options, [
+            'addedToBulk',
+            'can_edit',
+            'deleteUrl',
+            'description',
+            'downloadUrl',
+            'editUrl',
+            'emailedExport',
+            'exportType',
+            'formname',
+            'id',
+            'isDeid',
+            'lastBuildDuration',
+            'name',
+            'owner_username',
+            'sharing',
+        ], [
+            'case_type',
+            'isAutoRebuildEnabled',
+            'isDailySaved',
+            'isFeed',
+            'isOData',
+            'showLink',
+        ]);
```

Comment:
Careful readers may notice that this appears in previous diffs.  That's because github insists on using commit timestamps for ordering, even though that's not at all how git works.